ci_tests/src/index.ts

```diff
@@ -1,49 +1,122 @@
 import {
   CONTRACTS,
   ChainId,
+  ChainName,
   getEmitterAddressEth,
   getSignedVAAWithRetry,
   parseSequenceFromLogEth,
+  postVaaSolana,
 } from "@certusone/wormhole-sdk";
 import { WormholeRelayer__factory } from "@certusone/wormhole-sdk/lib/cjs/ethers-contracts";
 import {
   getDeliveryHashFromLog,
   getWormholeLog,
 } from "@certusone/wormhole-sdk/lib/cjs/relayer";
-import { Networkish } from "@ethersproject/networks";
+import { NodeWallet } from "@certusone/wormhole-sdk/lib/cjs/solana";
+import { PostedMessageData } from "@certusone/wormhole-sdk/lib/cjs/solana/wormhole";
+import { BN, web3 } from "@coral-xyz/anchor";
 import { NodeHttpTransport } from "@improbable-eng/grpc-web-node-http-transport";
-import { Wallet, getDefaultProvider, utils } from "ethers";
-import { existsSync, writeFileSync } from "fs";
+import * as spl from "@solana/spl-token";
+import {
+  BigNumber,
+  ContractReceipt,
+  ContractTransaction,
+  Wallet,
+  providers,
+  utils,
+} from "ethers";
 import { DummyTokenMintAndBurn__factory } from "../evm_binding/factories/DummyToken.sol/DummyTokenMintAndBurn__factory";
 import { DummyToken__factory } from "../evm_binding/factories/DummyToken.sol/DummyToken__factory";
 import { ERC1967Proxy__factory } from "../evm_binding/factories/ERC1967Proxy__factory";
 import { NttManager__factory } from "../evm_binding/factories/NttManager__factory";
 import { TransceiverStructs__factory } from "../evm_binding/factories/TransceiverStructs__factory";
 import { TrimmedAmountLib__factory } from "../evm_binding/factories/TrimmedAmount.sol/TrimmedAmountLib__factory";
 import { WormholeTransceiver__factory } from "../evm_binding/factories/WormholeTransceiver__factory";
+import { NTT } from "../solana_binding/ts/sdk";
```

Comment:
Right now the EVM bindings live in the ci_tests folder. I feel like this should be consistent between the bindings. So, maybe move the EVM bindings to the evm repo?

Reply:
addressed in #261